scanner/src/main/kotlin/Scanner.kt

```diff
@@ -86,7 +108,20 @@ fun scanOrtResult(
         }
 
     val scanResults = runBlocking {
-        scanner.scanPackages(packagesToScan, outputDirectory).mapKeys { it.key.id }
+        // Scan the projects from the ORT result.
+        val deferredProjectScan = async {
+            projectScanner.scanPackages(filteredProjectPackages, outputDirectory).mapKeys { it.key.id }
+        }
+
+        // Scan the packages from the ORT result.
+        val deferredPackageScan = async {
+            scanner.scanPackages(filteredPackages, outputDirectory).mapKeys { it.key.id }
+        }
+
+        val projectResults = deferredProjectScan.await()
+        val packageResults = deferredPackageScan.await()
+
+        projectResults + packageResults
```

Comment:
Here you are mixing the two results together ?! Didn't you say you were going to separate them in a different `ScannerRun` ?

Reply:
I don't remember whether I said "ScannerRun", but if so, that was wrong. We get separate `ScanResult`s, not `ScannerRun`s. In any case, the results are completely separate in the end, see this example result:

![image](https://user-images.githubusercontent.com/349154/131836231-6d2207db-be7b-45ae-8bf3-b47052baebe8.png)